src/meta/metainfo.go

```diff
@@ -312,9 +312,10 @@ func (i *info) AddToGlobalScopeNonLocked(filename string, sc *Scope) {
 }
 
 type FuncParam struct {
-	IsRef bool
-	Name  string
-	Typ   TypesMap
+	IsRef   bool
+	Name    string
+	Typ     TypesMap
+	Default *ConstValue
```

Comment:
Why can't it be a non-pointer type?
Zero value ConstValue has Type=Undefined, which is good enough to detect "no default value" case.

Reply:
Good idea. Done.